src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/NotifyClaimCallbackHandlerTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.unspec.handler.callback.user;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.sampledata.CallbackParamsBuilder;
+import uk.gov.hmcts.reform.unspec.sampledata.CaseDataBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.unspec.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.NOTIFY_DEFENDANT_OF_CLAIM;
+
+@SpringBootTest(classes = {
+    NotifyClaimCallbackHandler.class,
+    JacksonAutoConfiguration.class,
+    CaseDetailsConverter.class
+})
+class NotifyClaimCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private NotifyClaimCallbackHandler handler;
+
+    @Nested
+    class AboutToSubmit {
+
+        @Test
+        void shouldUpdateBusinessProcess_whenInvoked() {
+            CaseData caseData = CaseDataBuilder.builder().atStateAwaitingCaseNotification().build();
+            CallbackParams params = CallbackParamsBuilder.builder().of(CallbackType.ABOUT_TO_SUBMIT, caseData).build();
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("camundaEvent", "status")
+                .containsOnly(NOTIFY_DEFENDANT_OF_CLAIM.name(), "READY");
+        }
+    }
+
+    @Nested
+    class SubmittedCallback {
+
+        public static final String CONFIRMATION_BODY = "<br />What happens next\n\n"
+            + "The defendant legal representative's organisation has been notified and granted access to this "
+            + "claim.\n\n"
```

Comment:
Shouldn't we use `%n` instead of `\n` ?

Reply:
Hmm, seems we have places we use %n and places we use \n. I will leave as is, but maybe we should have tech debt to clean up?

Reply:
We did clean that few months back. Looks like it's time again to clean this one. :)